template/subscription_template.json

```diff
@@ -51,5 +51,9 @@
         "autoDeleteOnIdle": "P10675199DT2H48M5.4775807S"
       }
     }
+  ],
+  "dependsOn": [
+    "[resourceId('Microsoft.ServiceBus/namespaces', parameters('serviceBusNamespaceName'))]",
```

Comment:
Can we not use a single arm template to create a service Bus namespace with topic and subscription.

https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-resource-manager-namespace-topic